app/models/remote_theme.rb

```diff
@@ -419,7 +419,12 @@ def normalize_override(hex)
   end
 
   def update_theme_color_schemes(theme, schemes)
-    existing_schemes = ColorScheme.unscoped.where(theme_id: theme.id)
+    existing_schemes =
+      if theme.id
+        ColorScheme.unscoped.where(theme_id: theme.id)
+      else
+        []
```

Comment:
Late to the party, but if you want the same return type, which can be nice, you can do `ColorScheme.none` which will give you an empty AR relation.